geographic_msgs/msg/GeoPoseStamped.msg

```diff
@@ -0,0 +1,2 @@
+Header header
+geographic_msgs/GeoPose geo_pose
```

Comment:
I'm on the fence about this, but you could consider following the `position` naming already established in a few other messages:

https://github.com/ros-geographic-info/geographic_info/search?q=GeoPoint

OTOH, the convention from geometry_msgs is more toward just naming the field after the type, which would suggest either `geo_pose` or just `pose`.